.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Question
```

Comment:
I think this should be left out - we prefer to keep discussions and questions at the mailing list

Reply:
Makes sense.

Reply:
Removed.